### ISSUE TITLE
Setting the number of training iterations

### ISSUE THREAD

**Issue by Aks0121** [DeepLabCut/DeepLabCut#224]
**Your Operating system and DeepLabCut version**

macOS High Sierra, with an Anaconda Env, & DeepLabCut 2

**Describe the problem**

Due to hardware limitations, is there a way to limit the number of training iterations? 

**How to Reproduce the problem**

N/A

**Screenshots**

N/A

**Additional context**

N/A

**Comment by DonnieKim411** [comment]
You can modify how many iters to train on `pose_cfg.yaml` file under `dlc-models/your_iteration_num/your_shuffle_num/train`

In that config file, look at `multi_step`. That's where you modify the learning rate as well as number of iterations

**Comment by Aks0121** [comment]
Awesome. Thank you.

**Comment by MMathisLab** [comment]
But also training is a batch size of 1, so in terms of GPU size, frame size is your limitation, not iteration #s
